FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +30,30 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: self.url) { [weak self] result in
+			guard let _ = self else { return }
```

Comment:
Instead of using a `guard let` and ignoring the attribution, it's clearer to check for `nil`:
```suggestion
			guard self != nil else { return }
```